### ISSUE TITLE
External JS Breaks JS Combine?

### ISSUE THREAD

**Issue by SemperFiWS** [cubecart/v6#2012]
- 6.2.0 store
- stock 6.2.0 foundation skin
- Braintree plugin enabled

No JS file is created and added to 'head' section.

Error Log Entry:
```[10-May-2018 18:34:51 Europe/London] PHP Warning:  File /home/XXXXXX/public_html/<script src="https://js.braintreegateway.com/web/3.21.1/js/client.min.js"></script>
<script src="https://js.braintreegateway.com/web/3.21.1/js/hosted-fields.min.js"></script>
<script src="https://js.braintreegateway.com/web/3.21.1/js/paypal.min.js"></script>
<script src="https://js.braintreegateway.com/web/3.21.1/js/paypal-checkout.min.js"></script>
<script src="https://js.braintreegateway.com/web/3.21.1/js/data-collector.min.js"></script>
<script src="https://js.braintreegateway.com/web/3.21.1/js/three-d-secure.min.js"></script>
<script src="https://www.paypalobjects.com/api/checkout.js"></script> 
does not exist! in /home/XXXXXX/public_html/includes/lib/smarty/plugins/function.combine.php on line 159```

Set Braintree to disabled, all works correctly again.

**Comment by bhsmither** [comment]
Probably related to #2008, but I think there is a better way.

Check for src that is not `{$STORE_URL}`. Array those separately, still squeeze the lines, append to what was already combined.

**Comment by SemperFiWS** [comment]
Looking at the fix for #2008 it will cause issues when scripts included are depending on jquery having been loaded already.

Moving this:

```{foreach from=$HEAD_JS item=js}{$js}{/foreach}```

below the combine would fix that issue.

**Comment by abrookbanks** [comment]
https://github.com/cubecart/v6/issues/2008
